pom.xml

```diff
@@ -4,32 +4,206 @@
+            <showWarnings>true</showWarnings> <compilerArguments> <Xmaxwarns>10000</Xmaxwarns> <Xlint/> </compilerArguments> <encoding>UTF-8</encoding> </configuration> </plugin> -->
+
+            <!-- handle unit testing of java classes -->
+            <!-- <plugin> <groupId>org.apache.maven.plugins</groupId> <artifactId>maven-surefire-plugin</artifactId> <version>2.18.1</version> <executions> <execution> <id>default-test</id> <goals> <goal>test</goal> </goals> </execution> </executions>
+            <configuration> <includes> <include>**/Test*.java</include> <include>**/*Test.java</include> </includes> <excludes> <exclude>**/*FuncTest.java</exclude> </excludes> <forkCount>1</forkCount> <reuseForks>false</reuseForks> <systemPropertyVariables>
+            <environment>${environment}</environment> <site>${site}</site> </systemPropertyVariables> </configuration> </plugin> -->
+
+            <!-- log4j audit maven plugin -->
+            <plugin>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-audit-maven-plugin</artifactId>
+                <version>${log4j-audit.version}</version>
+                <executions>
+                    <execution>
+                        <id>generate</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <packageName>org.apache.logging.log4j.audit.event</packageName>
+                            <enterpriseId>18060</enterpriseId>
+                            <maxKeyLength>32</maxKeyLength>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <!-- versions-maven-plugin -->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>versions-maven-plugin</artifactId>
+                <version>${versions-maven-plugin.version}</version>
+                <configuration>
+                    <generateBackupPoms>false</generateBackupPoms>
+                </configuration>
+            </plugin>
+            <!-- maven enforcer and complier -->
+            <!-- <plugin> <groupId>org.apache.maven.plugins</groupId> <artifactId>maven-compiler-plugin</artifactId> <version>3.8.0</version> <configuration> <source>[1.8,)</source> <target>[1.8,)</target> <showDeprecation>true</showDeprecation>
+            <showWarnings>true</showWarnings> <compilerArguments> <Xmaxwarns>10000</Xmaxwarns> <Xlint/> </compilerArguments> <encoding>UTF-8</encoding> </configuration> </plugin> -->
+            <plugin>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>${maven-enforcer-plugin.version}</version>
```

Comment:
Don't need to redefine this version, same reason as in other plugins defined in parent.